tests/conftest.py

```diff
@@ -653,32 +528,43 @@ def network_node(
     oef_addr, oef_port, pytestconfig, timeout: float = 2.0, max_attempts: int = 10
 ):
     """Network node initialization."""
-    if sys.version_info < (3, 7):
-        pytest.skip("Python version < 3.7 not supported by the OEF.")
-        return
+    client = docker.from_env()
+    image = OEFSearchDockerImage(client, oef_addr, oef_port)
+    yield from _launch_image(image, timeout, max_attempts)
 
-    if os.name == "nt":
-        pytest.skip("Skip test as it doesn't work on Windows.")
 
-    _stop_oef_search_images()
-    c = _create_oef_docker_image(oef_addr, oef_port)
-    c.start()
+@pytest.fixture(scope="session")
+def ganache(ganache_addr, ganache_port, timeout: float = 2.0, max_attempts: int = 10):
+    """Launch the Ganache image."""
+    client = docker.from_env()
+    image = GanacheDockerImage(client, "http://127.0.0.1", 8545)
+    yield from _launch_image(image, timeout=timeout, max_attempts=max_attempts)
 
-    # wait for the setup...
-    logger.info("Setting up the OEF node...")
-    success = _wait_for_oef(max_attempts=max_attempts, sleep_rate=timeout)
 
+def _launch_image(image: DockerImage, timeout: float = 2.0, max_attempts: int = 10):
+    """
+    Launch image.
+
+    :param image: an instancoe of Docker image.
+    :return: None
+    """
+    image.check_skip()
```

Comment:
I think the instantiation needs to happen after skip! Skip needs to be class method independent of docker